### ISSUE TITLE
Unable to Search

### ISSUE THREAD

**Issue by kirannunnak** [AnupKumarPanwar/Uber-Like-Cab-Service#19]
App when installed in mobile after all development changes like firebase, google api configuration, doesnt save driver or customer info, or when clicked google search, the container just comes and goes down without proper search functionality?Can u let me know the fix?also if we search and if driver enables "working" , it will show drivers available?

**Comment by AnupKumarPanwar** [comment]
Yes actually the Placepicker API got deprecated. That's why the map is crashing. I'll update soon.

**Comment by HMJCPS** [comment]
please upgrade
